### DISCUSSION TITLE
How to remove it

### DISCUSSION THREAD

**Discussion by StendUper** [ruffle-rs/ruffle#5190] [Q&A]
i have some issues when using Ruffle in some flash games and i just install flash back on my pc with some manipulation  (working now) but ruffle still open all swf files in browser and idk how to remove it.

**Comment by ROBERT-MCDOWELL** [comment]
just remove the extension or the JS and wasm file

**Comment by n0samu** [comment]
See also: https://github.com/ruffle-rs/ruffle/wiki/Frequently-Asked-Questions-For-Users/#how-do-i-uninstall-ruffle

**Comment by HugoEspinosa8005** [comment]
ruffle is infected in my browser and edge chrome I want to remove it please can I solve it?
